src/pages/ExamsPage/components/ExternalReviewDashboard.test.jsx

```diff
@@ -18,10 +18,14 @@ describe('ExternalReviewDashboard', () => {
   it('does not include iframe if no exam provided', () => {
     render(<ExternalReviewDashboard exam={null} />);
     expect(screen.queryByTitle('lti_link')).not.toBeInTheDocument();
+    expect(screen.queryByText('Open the Review Dashboard for Test Proctored Exam')).not.toBeInTheDocument();
```

Comment:
I just noticed the name of this test, 'does not include iframe if no exam provided', isn't really accurate anymore